### DISCUSSION TITLE
Show sponsors?

### DISCUSSION THREAD

**Discussion by jankapunkt** [node-oauth/node-oauth2-server#183] [General]
Hi @HappyZombies @jorenvandeweyer I would like to show our sponsoring options in the repos. What do you think?

**Comment by HappyZombies** [comment]
I personally have never done a sponsor thing in GitHub before, what are your goals and expectations of having this enabled in the project?

**Reply by jankapunkt** [reply]
Sometimes people simply say "thank you" by sponsoring a few bucks which is really a nice compensation for otherwise free work. Sponsoring can also be an incentive to put more focus / effort into a specific package/project. I don't think I will ever generate a stable income from open source work but I like the idea to get something back from time to time.

**Reply by jorenvandeweyer** [reply]
If you want you can enable it, it could be a good incentive to put more effort into the package.

**Reply by jankapunkt** [reply]
Ok I will add them for now and close this discussion. Feel free to reopen if this goes into the wrong direction or we want to change this.

**Reply by HappyZombies** [reply]
Yup I am good with this 👍
